netlink-packet-route/src/rtnl/route/nlas/next_hops.rs

```diff
@@ -55,9 +57,15 @@ impl<T: AsRef<[u8]>> NextHopBuffer<T> {
     }
 }
 
+impl<'a, T: AsRef<[u8]> + ?Sized> NextHopBuffer<&'a T> {
+    pub fn nlas(&self) -> impl Iterator<Item = Result<NlaBuffer<&'a [u8]>, DecodeError>> {
+        NlasIterator::new(&self.gateway_nla()[..(self.length() as usize - GATEWAY_OFFSET)])
+    }
+}
+
 const GATEWAY_OFFSET: usize = 8;
```

Comment:
Not sure why I put this constant here, would you mind moving it up, above the `buffer!` call where it's first used?

Reply:
Sure, done.